content/docs/user-guide/networking/multiplayer/split.md

```diff
@@ -0,0 +1,224 @@
+---
+title: Client and Server Launchers with the Multiplayer Gem
+description: Build Open 3D Engine launchers with the Multiplayer Gem specifically targeting Clients, Servers or both.
+linkTitle: Separating Client and Server
+---
+
+The Multiplayer Gem supports builds that contain only client logic, only server logic, or both client and server logic. This allows users to create executables of smaller size by excluding unnecessary logic and dependencies. It also allows hiding potentially sensitive logic unique to one executable from the other. For example, a publicly released game client that doesn't ship with server logic can make hacking the simulation more difficult.
+
+The splitting functionality produces multiple build types.
+* GameLauncher is a client only launcher
+* ServerLauncher is a server only launcher suitable for Dedicated Servers
+* UnifiedLauncher provides both and is suitable for Client Hosted Servers
+
+This functionality is implemented through a variety of build mechanisms and it's important to understand these mechanisms in any Gem or project using the Multiplayer Gem.
+
+## Splitting Up Client and Server logic
+
+The Multiplayer Gem contains code files that can be divided into two categories.
+1. Files that are fully required on all launcher types.
+2. Files that have parts conditionally compiled out depending on launcher type and their dependents.
+
+These file lists are maintained in multiplayer_files.cmake and multiplayer_split_files.cmake respectively.
```

Comment:
```suggestion
These file lists are maintained in `multiplayer_files.cmake` and `multiplayer_split_files.cmake` respectively.
```

Would a developer call these files differently? Or are these specific to the Multiplayer Gem?

Reply:
They are specific to the Multiplayer Gem and would only be a consideration for a developer if they are specifically modifying the Multiplayer Gem. Otherwise they would use one of Multiplayer's targets (Common, Client, Server or Unified).